Sources/XCHammer/XcodeTarget.swift

```diff
@@ -454,10 +454,13 @@ public class XcodeTarget: Hashable, Equatable {
             return xcTargetName + ".xctest"
             case .AppExtension, .XPCService, .Watch1App, .Watch2App, .Watch1Extension, .Watch2Extension, .TVAppExtension:
             return xcTargetName + ".appex"
-            case .StaticLibrary:
+            case .StaticLibrary, .Tool:
             return xcTargetName
+            case .Bundle:
```

Comment:
Do we actually need this one? I thought bundles weren't built

Reply:
I actually hit this case